app/models/route_stop_pattern.rb

```diff
@@ -185,10 +185,12 @@ def tl_geometry(stop_points, issues)
     if issues.include?(:has_before_stop)
       points = self.geometry[:coordinates].unshift(stop_points[0])
       self.geometry = RouteStopPattern.line_string(points)
+      self.is_modified = true
```

Comment:
do these two additions belong in these documentation PR? or maybe they accidentally came over from another branch in progress?